Source/ThirdParty/o3dgc.js

```diff
@@ -1,4 +1,4 @@
-define([], function() {
+define(['../Core/getTimeStamp'], function(getTimeStamp) {
```

Comment:
@kainino0x instead of basing this on our function, can we instead open a pull request for `o3dgc` that includes our function locally: https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Source/Core/getTimestamp.js